### PR TITLE
Ran spell checker through all man pages

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: libpod/container_api.go [hunk 1]
```diff
@@ -262,6 +262,9 @@ func (c *Container) Init() (err error) {
 			return err
 		}
 	}
+	if err := c.setupOCIHooks(&g); err != nil {
```
Comment (mheon) [inline libpod/container_api.go]: I think this snuck in from another of your PRs

**Comment by mheon** [comment]
A bit of the hooks PR was transplanted in here but otherwise LGTM

File: libpod/container_api.go (hunk 1 above)
Comment (rhatdan) [reply libpod/container_api.go]: oops

**Comment by mheon** [comment]
bot, retest this please

**Comment by baude** [comment]
bot, retest this please

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

**Comment by rhatdan** [comment]
@rh-atomic-bot r=TomSweeneyRedHat

**Pull request closed without merging**